interbtc-spec/docs/source/spec/nomination.rst

```diff
@@ -5,84 +5,61 @@ Vault Nomination
-
-#. Liquidation slashing is handled as follows.
-
-   #. In case the collateral managed by the Operator falls below the liquidation threshold, the Operator and Nominators are slashed proportionally to their collateral.
-   
-   #. In case the Operator steals Bitcoin deposited at its address, its collateral is used to cover as much of the slashed amount as possible. If the Operator’s collateral was not enough to cover the entire amount, the Nominators are slashed proportionally for the remaining amount.
-
-#. Collateral withdrawals are first requested and then executed. A withdrawal request:
-
-   #. Decreases the issuable interbtc capacity.
-   
-   #. May be cancelled if not executed. The amount in the cancelled withdrawal request becomes backing collateral again.
-   
-   #. Is subject to a window of delay (unbonding period) that allows Nominators and the Operator to react.
+#. Vaults must choose to opt in to the Nomination protocol.
+#. Nominators select a Vault to which they can delegate collateral. They will earn a fraction of any rewards generated by the Vault.
+#. Vault replacement is disallowed with nominated collateral. Otherwise, Security Assumptions 1 and 2 would be violated.
+#. The nominated collateral:
 
-      #. Operator Unbonding Period. This window is longer, because Nominators are assumed to be mostly offline.
+   #. Cannot be withdrawn by the Vault,
+   #. Is locked on the parachain,
+   #. Is capped at a fraction of the Vault’s deposited collateral (:ref:`maxNominationRatio`). 
       
-      #. Nominator Unbonding Period. This window is shorter, because the Vault Operator is assumed to always be online.
+#. Liquidation slashing is handled as follows:
 
-#. Collateral withdrawals are subject to the following restrictions.
+   #. In case the collateral managed by the Vault falls below the liquidation threshold, the Vault and Nominators are slashed proportionally to their collateral.
+   #. In case the Vault steals Bitcoin deposited at its address, the Vault and Nominators are slashed proportionally to their collateral.
 
-   #. The remaining collateralization of an operator must not be below the secure collateral threshold.
-   
-   #. Operator withdrawals must not cause nominated collateral to exceed the Max Nomination Ratio.
+#. Vaults may opt out of the Nomination protocol which force refunds Nominators. 
 
-#. Forced collateral withdrawal. If an operator’s withdrawal would result in a violation of the Max Nomination Ratio, automatically refund excess nominated collateral to the nominators, proportionally. Both the operator withdrawal and the nominator refunds are subject to the unbonding period.
-#. If an Operator issued zero interbtc, it can deregister and automatically refund Nominators their collateral.
-#. When an Operator is banned, its collateralization is lowered to the secure collateral threshold by automatically refunding nominated DOT, proportionally.
+.. _maxNominationRatio:
```

Comment:
Should this already go into the data model section?

Reply:
I think this is more of the general concept, the functionality is outlined in a section below.

Reply:
Added the ratio definition above so that the reader does not have to click two times to find it :)